test/test.py

```diff
@@ -46,12 +47,47 @@ def setUp(self):
         # Suppress `sys:1: ResourceWarning: unclosed <ssl.SSLSocket fd=6, family=AddressFamily.AF_INET, ...`
         warnings.simplefilter("ignore", ResourceWarning)
 
-    def _rename_keys(self, d, **kwargs):
-        for new_name, old_name in kwargs.items():
-            assert new_name != old_name
-            if old_name in d:
-                d[new_name] = d[old_name]
-                del d[old_name]
+    # def _rename_keys(self, d, **kwargs):
+    #     for new_name, old_name in kwargs.items():
+    #         assert new_name != old_name
+    #         if old_name in d:
+    #             d[new_name] = d[old_name]
+    #             del d[old_name]
+
+    def test_get_jsonpath(self):
+        test_donor: dict = {
+            "diseases": [
+                {
+                    "text": "some_disease"
+                },
+                {
+                    "text": "some_other_disease"
+                }
+            ],
+            "species": {
+                "taxon_id": "9606"
+            }
+        }
+
+        diseases_path = "diseases.text"
+        donor_diseases = _get_path(test_donor, diseases_path)
+        self.assertTrue(donor_diseases == ["some_disease", "some_other_disease"])
+
+        taxon_id_path = "species.taxon_id"
+        taxon_id = _get_path(test_donor, taxon_id_path)
+        self.assertTrue(taxon_id == "9606")
+
+
+    def test_ontology_label(self):
+        ontology = {'text': 'foo', 'ontology': 'bar', 'ontology_label': 'baz',}
```

Comment:
`ontology_label` already has doctests and those are run by the Travis build. Please move any cases from here that  are not covered in the doctest to the doctest and remove this test.